crates/sui-indexer/migrations_v2/2023-08-19-044020_events/up.sql

```diff
@@ -18,7 +18,7 @@ CREATE TABLE events
     PRIMARY KEY(tx_sequence_number, event_sequence_number)
 );
 
-CREATE INDEX events_senders ON events USING GIN(senders);
-CREATE INDEX events_package_module ON events (package, module);
-CREATE INDEX events_event_type ON events (event_type);
+CREATE INDEX events_package ON events (package, tx_sequence_number, event_sequence_number);
```

Comment:
almost right! In our case it will be 
```
where package = A and tx_sequence > B
```
and 
```
to double check so now, this index will be used in all queries:
where package = A
where package = A and tx_sequence > B
where package = A and tx_sequence = B and event_sequence > C?
```